docs/building/MlNetMklDeps/MlNetMklDeps.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
```

Comment:
Do you think it would be a good idea to put a README in this directory as well? The README could have the instructions for how to build the package. Then it would all be out in the open, and we wouldn't have to go looking for the instructions.